src/Lean/Elab/Command.lean

```diff
@@ -271,13 +277,14 @@ partial def elabCommand (stx : Syntax) : CommandElabM Unit := do
       else do
         trace `Elab.command fun _ => stx;
         let s ← get
-        let stxNew? ← catchInternalId unsupportedSyntaxExceptionId
-          (do let newStx ← adaptMacro (getMacros s.env) stx; pure (some newStx))
-          (fun ex => pure none)
-        match stxNew? with
-        | some stxNew => withMacroExpansion stx stxNew <| elabCommand stxNew
+        match ← liftMacroM <| expandMacroImpl? s.env stx with
```

Comment:
Could you please use `()` around `← liftMacroM <| expandMacroImpl? s.env stx`? I do that in all code that I write/maintain.

Reply:
Ok, noted. This specific one was easy to update via `sed` fortunately, so I did so everywhere.